spec/lib/interferon_spec.rb

```diff
@@ -7,6 +7,11 @@
 describe Interferon::Destinations::Datadog do
   let(:the_existing_alerts) { mock_existing_alerts }
   let(:dest) { MockDest.new(the_existing_alerts) }
+  let(:datadog) do
+    Interferon::Destinations::Datadog.new(
+      'app_key' => 'TEST_APP_KEY', 'api_key' => 'TEST_API_KEY'
+    )
+  end
 
   shared_examples_for 'alert_option' do |alert_option, same_value, different_value, alert_dsl|
     let(:json_message) { 'message' + "\n#{Interferon::Destinations::Datadog::ALERT_KEY}" }
```

Comment:
Should replace `Interferon::Destinations::Datadog::ALERT_KEY` with `datadog.alert_key` in the rest of the file.